testData/solutions/win32.json5

```diff
@@ -5807,6 +6472,458 @@
+                            },
+                            "noMatch": 0
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": "pvParam"
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "transform": [
+                        {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "ScrollFocusRoutingConfig.value",
+                            "defaultOutputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollRouting",
+                            "match": {
+                                "0": "RoutingFocus",
+                                "1": "RoutingHybrid"
+                            },
+                            "noMatch": "RoutingFocus"
+                        }
+                    ]
+                }
+            },
+            "configureMouseCursorShadow": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETCURSORSHADOW",
+                    "setAction": "SPI_SETCURSORSHADOW",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "BOOL"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "MouseCursorShadowConfig": {
+                        "schema": {
+                            "title": "Enable disable mouse cursor shadow",
+                            "description": "Changes the way mouse scrolling is passed to applications",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "MouseCursorShadowConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.EnableCursorShadow",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": "pvParam"
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
```

Comment:
It should be possible to remove this inverse as the system can compute it easily

Reply:
Doing it and testing.

Reply:
Tests failed for this, could be possible that they are indeed necessary?